### PR TITLE
Use JWT to link directly to Admin Portal

### PR DESCRIPTION
**Pull request by willmanduffy**
* Previously, the Admin Portal demo had some slowness due to the fact
  that we were setting up data when a user clicked "Configure SSO".
  With this PR we are instead eager loading that data on page load and
  then creating a JWT and redirecting directly to the Admin Portal with
  it.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+WORKOS_ADMIN_PORTAL_HOSTNAME=setup.workos-test.com
```
Comment (willmanduffy) [inline .env.example:1]: Adding environmental variable support so that we can setup a staging environment.

File: app.rb [hunk 2]
```diff
@@ -38,32 +52,13 @@
...
-
-    portal_link = WorkOS::Portal.generate_link(
-     intent: 'sso',
-     organization: organization.id
-    )
-
-    redirect portal_link
-  else
-    organization = organizations.data.first
+  payload = {
```
Comment (willmanduffy) [inline app.rb:55]: All of this will be inside of the Ruby SDK eventually, but we're holding off on that for now.

File: app.rb [hunk 3]
```diff
@@ -23,6 +27,16 @@
   company_name = params['company'] || "demo-#{Time.now.to_i}"
   domain = company_name + '.com'
 
+  organizations = WorkOS::Portal.list_organizations(
```
Comment (willmanduffy) [inline app.rb:30]: Moved this logic from /portal to be loaded when we load the demo. I'd prefer to have the fake demo app have an extra moment to prepare data than to make our Admin Portal itself seem slow. 

It does have the side effect of creating the demo org earlier, but I'm not super worried about that.

Soon we'll be rebuilding the demo to contain multiple demos and I'll refactor this to create the data when the demo is selected with a loading spinner if applicable.

**Review by maxchehab — APPROVED** [review]

File: app.rb [hunk 4]
```diff
@@ -2,10 +2,14 @@
 
 require 'sinatra'
 require 'sinatra/reloader' if development?
-require 'workos'
+require 'dotenv/load' if development?
```
Comment (marktran) [inline app.rb]: Can we can remove the `if development?` bit? There aren't any .env files in the repo and it's sometimes desirable to run your development setup in different environments.

**Review by marktran — APPROVED** [review]
Minor comment, looks good overall

**Pull request merged**